src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -1249,6 +1281,14 @@ def call(
             attentions=outputs.attentions,
         )
 
+    def serving_output(self, output):
+        return TFTokenClassifierOutput(
+            loss=None,
```

Comment:
I'm assuming all losses are set to `None` because no training happen in `TFServing`, right?

Reply:
Yes, no training through TFServing, then loss to `None` to make it disappear from the future output.

Reply:
Since it's the default in all `ModelOutput`, why not omit it entirely?